pkg/agent/storage/bbolt/keyvalue_store_bbolt.go

```diff
@@ -214,14 +215,32 @@ func (s *bboltKeyValueStore) Update(data io.Reader) error {
 // SetupDB is used for creating bbolt databases for testing
 func SetupDB(t *testing.T) *bbolt.DB {
```

Comment:
I believe this will cause the testing.T package to be included in our compiled binary. I suggested accepting a filepath as a parameter instead. Leave it up to the caller to use `SetupDb(t.TempDir())`

Reply:
Ooh good catch! I see another spot in the codebase we've done a similar pattern. I go fix!

Reply:
I ended up moving this function to the `storageci` package which is only used by tests and keeps it from being compiled into our release binary.